common/src/main/scala/ClassTwoAndThreeResult.scala

```diff
@@ -41,12 +41,18 @@ case class ClassTwo(
   hrpDate: Option[LocalDate],
   finalDate: Option[LocalDate],
   noOfWeeks: Int = 52,
-  qualifyingRate: BigDecimal
+  qualifyingRate: BigDecimal,
+  lel: BigDecimal
 ) extends ClassTwoOrThree {
   def rate = weeklyRate.default
-  def lowerEarningLimit = ((qualifyingRate + 50) / noOfWeeks) gives
-    s"lowerEarningLimit: (qualifyingRate + 50) / noOfWeeks = ($qualifyingRate + 50) / $noOfWeeks"
+  // def lowerEarningLimit = lel match {
```

Comment:
are you wanting to keep this for some reason or it is safe to delete?

Reply:
I've taken this out